cachedHTTP/README.md

```diff
@@ -0,0 +1,216 @@
+# cachedHTTP
```

Comment:
I'm not seeing the implementation for cachedHTTP in this PR? Am I missing something or was it not checked in?

Reply:
Ok, now that I'm reading further in the PR I'm getting that the purpose is to create a `uv` environment that the ts source in `scripts/` can manipulate.

I think we should move this directory into `scripts/` and colocate it with the scripts that actually use it, rather than having it at the top level.

Reply:
done